optimize/minimize.go

```diff
@@ -70,6 +85,13 @@ type Method interface {
 	// composed to specify the valid fields. Methods are free to use or
 	// ignore these values.
 	//
+	// Successful execution of an Operation typically requires the Method to
+	// modify the X field of Location, and MajorIteration operations may use
+	// the information in the remaining fields. Methods are encouraged to leave
+	// the remaining fields untouched so that the memory may be re-used by the
+	// problem. Methods should not allocate memory themselves for a Location,
+	// instead the memory should be left untouched or set to nil.
```

Comment:
Is the repeated "untouched" intentional or is it an editing artifact?

Reply:
Made better comment.